tests/Dockerfile.daemon

```diff
@@ -48,6 +48,7 @@ COPY --from=build /mender-install/usr/share/mender /usr/share/mender
 COPY --from=build /mender-install/usr/share/doc/mender-auth /usr/share/doc/mender-auth
 COPY --from=build /mender-install/lib/systemd/system/mender-updated.service /lib/systemd/system/mender-updated.service
 COPY --from=build /mender-install/lib/systemd/system/mender-authd.service /lib/systemd/system/mender-authd.service
+COPY --from=build /mender-install/lib/systemd/system/mender-client-data-dir.service /lib/systemd/system/mender-client-data-dir.service
```

Comment:
This we don't need.

On the one hand we are not enabling the option (hence it is not installed), but on the other hand this container has no data partition so we don't want the service.

Reply:
Great! Not a problem and fixed!